### PR TITLE
blog: Using webcam with node-red

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/blog/2024/02/using_webcam_with_node-red.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+title: Using webcam with Node-Red
```
[the remark this answers is not in the document]
Comment (sumitshinde-84) [reply src/blog/2024/02/using_webcam_with_node-red.md]: okay zj i will !

File: src/blog/2024/02/using_webcam_with_node-red.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+title: Using a webcam with Node-RED
+subtitle: Guide to integrating webcam features into Node-RED applications.
+description: Guide to integrating webcam features into Node-RED applications.
+date:
```
Comment (ZJvandeWeg) [inline src/blog/2024/02/using_webcam_with_node-red.md:5]: @sumitshinde-84 Can you move this file, I think we should target the 1st of March.

```suggestion
date: 2024-03-01
```

File: src/blog/2024/02/using_webcam_with_node-red.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
+---
+title: Using a webcam with Node-RED
+subtitle: Guide to integrating webcam features into Node-RED applications.
+description: Guide to integrating webcam features into Node-RED applications.
+date: 
+authors: ["sumit-shinde"]
+image: /blog/2024/02/images/using_webcam_with_node-red_capturing_images_by_passing_payload.gif
```
Comment (ZJvandeWeg) [inline src/blog/2024/02/using_webcam_with_node-red.md:7]: @Yndira-FlowForge Can you make a better tile and have Sumit upload it? (He's using a fork, making it hard for you to push it directly)
```suggestion
image: /blog/2024/02/images/using_webcam_with_node-red_capturing_images_by_passing_payload.gif
```

File: src/blog/2024/02/using_webcam_with_node-red.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+2. Initially, you'll see a black interface with a power button on the dashboard. Clicking this button will activate the webcam. Ensure that you grant permission to the dashboard to access your webcam.
+3. Once the webcam is active, you can capture images by clicking the button with the camera icon located at the bottom center of the webcam interface.
+4. The widget returns a Base64 string containing the captured image in PNG format.
+
+!["capturing images using webcam widget controls"](./images/using_webcam_with_node-red_capturing_images_by_control.gif "capturing images using webcam widget controls")
+
+## Capturing images by passing payload
+
+1. Drag an inject node onto the canvas.
+2. Set msg.payload as a 'capture' string. You can also set an interval time to automatically send the message after a specific interval, or you can keep it unchanged to manually send the payload by clicking the inject button.
```
Comment (ZJvandeWeg) [inline src/blog/2024/02/using_webcam_with_node-red.md:51]:
```suggestion
2. Set `msg.payload` to `capture` as string. You can also set an interval time to automatically send the message after a specific interval, or you can keep it unchanged to manually send the payload by clicking the inject button.
```

File: src/blog/2024/02/using_webcam_with_node-red.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+3. Connect the output of the inject node to the input of the webcam widget.
+4. Deploy the flow.
+
+By passing the "capture" string as payload, the webcam widget will activate (if it's off) and capture images automatically, without requiring user interaction. This method is commonly used in industrial applications which commonly depend on automated actions.
```
Comment (ZJvandeWeg) [inline src/blog/2024/02/using_webcam_with_node-red.md:55]: Using commonly twice :) 
```suggestion
By passing the "capture" string as payload, the webcam widget will activate (if it's off) and capture images automatically, without requiring user interaction. This method is commonly used in industrial applications which depend on automated actions.
```

File: src/blog/2024/02/using_webcam_with_node-red.md (hunk 2 above)
Comment (sumitshinde-84) [reply src/blog/2024/02/using_webcam_with_node-red.md]: Sure, ZJ. I'll create a new folder for the third month and move all the files into it

File: src/blog/2024/02/using_webcam_with_node-red.md (hunk 3 above)
Comment (Yndira-FlowForge) [reply src/blog/2024/02/using_webcam_with_node-red.md]: @ZJvandeWeg @sumitshinde-84 here it is
Using-a-webcam-with-Node-RED

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**